lib/ansible/modules/cloud/docker/docker_volume.py

```diff
@@ -257,6 +259,9 @@ def main():
         supports_check_mode=True
     )
 
+    if (DOCKER_VERSION is not None) and (LooseVersion(DOCKER_VERSION) < LooseVersion('1.10.0')):
+        client.module.fail_json(msg='Minimal required version for Docker-py is 1.10.0')
```

Comment:
Could you please change `Docker-py` to `docker-py`? That's the spelling used in all other docker modules.